### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Notebooks - Create_Keyboard: The user is unable to move out of the '+Note' edit pad to access other associated controls present in the UI using keyboard tab or shift tab keys.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19874]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#Keyboard;#AINotInScope;#WCAG2.1.1;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Tab to 'File' menu.
5. Tab to 'New Notebook' and select.
6. Tab to '+Text' button and select.
8. Use keyboard to move out of the edit pad.

## Actual:
The user is unable to move out of the '+Note' edit pad to access other associated controls present in the UI using keyboard tab or shift tab keys.

## Expected:
The user should be able to move out of the '+Note' edit pad to access other associated controls present in the using keyboard tab, shift tab or shortcut keys.

## User Impact:
Keyboard users will face difficulty in accessing the controls in the UI if they are not accessible using regular keyboard interactive keys.

## Attachment:
![13_Unable to get](https://user-images.githubusercontent.com/98393966/176432671-46b11f44-91d5-4f78-9955-ba4138a1f5c5.gif)

**Comment by chlafreniere** [comment]
@bbarik06 if a user presses Escape, the user will no longer be in edit mode. After that point, users will be able to navigate throughout the rest of the notebook.

shift+tab is valid in the WYSIWYG editor to outdent the current selection.

**Comment by alanrenmsft** [comment]
this is by design, since we haven't heard back for so long, I am going to close the issue.

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.36.1
![image](https://user-images.githubusercontent.com/85863999/219282231-08c8c3b4-d502-4ad4-a6db-f586fd865512.png)
